sorbet/rbi/gems/ast@2.4.2.rbi

```diff
@@ -0,0 +1,584 @@
+# typed: true
```

Comment:
do all these actually need to be committed?

Reply:
Yep
https://sorbet.org/docs/adopting#step-6-source-control